ipaserver/install/server/upgrade.py

```diff
@@ -1469,30 +1491,44 @@ def uninstall_dogtag_9(ds, http):
     http.restart()
 
 
-def mask_named_regular():
-    """Disable named, we need to run only named-pkcs11, running both named and
-    named-pkcs can cause unexpected errors"""
-    if sysupgrade.get_upgrade_state('dns', 'regular_named_masked'):
+def named_switch_service():
```

Comment:
A test is failing due to an error in this function:

```
2020-04-29T06:57:55Z ERROR IPA server upgrade failed: Inspect /var/log/ipaupgrade.log and run command ipa-server-upgrade manually.
2020-04-29T06:57:55Z DEBUG   File "/usr/lib/python3.7/site-packages/ipapython/admintool.py", line 179, in execute
    return_value = self.run()
  File "/usr/lib/python3.7/site-packages/ipaserver/install/ipa_server_upgrade.py", line 54, in run
    server.upgrade()
  File "/usr/lib/python3.7/site-packages/ipaserver/install/server/upgrade.py", line 2407, in upgrade
    upgrade_configuration()
  File "/usr/lib/python3.7/site-packages/ipaserver/install/server/upgrade.py", line 2206, in upgrade_configuration
    named_switch_service(),
  File "/usr/lib/python3.7/site-packages/ipaserver/install/server/upgrade.py", line 1526, in named_switch_service
    bind.setup_service()
  File "/usr/lib/python3.7/site-packages/ipaserver/install/bindinstance.py", line 800, in setup_service
    self.ldap_configure('DNS', self.fqdn, None, self.suffix)
  File "/usr/lib/python3.7/site-packages/ipaserver/install/service.py", line 654, in ldap_configure
    CONFIGURED_SERVICE, name, fqdn, ldap_suffix, config
  File "/usr/lib/python3.7/site-packages/ipaserver/install/service.py", line 671, in _ldap_enable
    post_add_config=extra_config_opts)
  File "/usr/lib/python3.7/site-packages/ipaserver/install/service.py", line 177, in set_service_entry_config
    api.Backend.ldap2.add_entry(entry)
  File "/usr/lib/python3.7/site-packages/ipapython/ipaldap.py", line 1648, in add_entry
    self.conn.add_s(str(entry.dn), list(attrs.items()))
  File "/usr/lib64/python3.7/contextlib.py", line 130, in __exit__
    self.gen.throw(type, value, traceback)
  File "/usr/lib/python3.7/site-packages/ipapython/ipaldap.py", line 1086, in error_handler
    raise errors.NotFound(reason=arg_desc or 'no such entry')

2020-04-29T06:57:55Z DEBUG The ipa-server-upgrade command failed, exception: NotFound: no such entry
2020-04-29T06:57:55Z ERROR Unexpected error - see /var/log/ipaupgrade.log for details:
```

Reply:
Fixed